ansible/roles/dev-desktop/files/scripts/init_git.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+import requests
+import re
+import subprocess
+import getpass
+
+for cfg in ["email", "name"]:
+    if subprocess.run(["git", "config", "--global", f"user.{cfg}"], stdout=subprocess.DEVNULL).returncode == 0:
+        print("git username and email already configured")
+        exit(0)
+
+team = requests.get("https://raw.githubusercontent.com/rust-lang/team/gh-pages/v1/teams/cloud-compute.json")
+team.raise_for_status()
+
+username = getpass.getuser()
+username = username.lstrip("gh-")
+
+print(f"found github name: {username}")
+
+for member in team.json()["members"]:
+    github = member["github"]
+    name = member["name"]
+    if github == username:
+        profile = f"https://raw.githubusercontent.com/rust-lang/team/master/people/{github}.toml"
```

Comment:
Hmm, rather than querying the file we should have a new team-api.infra.rust-lang.org endpoint to get the information we need.